src/utils/__tests__/hasInterpolation-test.js

```diff
@@ -0,0 +1,20 @@
+import hasInterpolation from "../hasInterpolation"
+import test from "tape"
+
+test("hasInterpolation", t => {
+  t.ok(hasInterpolation("(min-width: 10px)"), "has no interpolation")
```

Comment:
Your `ok` and `notOK` are backwards.

They should match the test description i.e. test("hasInterpolation"), so "ok" means the string has interpolation.